src/api/models/problem.model.ts

```diff
@@ -85,4 +109,6 @@ const problemModel = new Schema(
   { collection: '_problem' }
 );
 
-export default mongoose.model('problemModel', problemModel);
+const ProblemModel = model('problemModel', problemSchema);
```

Comment:
I split lines 112 and 114 up because TypeScript was unhappy with `export default model('problemModel', problemSchema)` for some reason

Reply:
yeah that's fine with me